tsconfig.json

```diff
@@ -14,7 +14,9 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "plugins": [{ "name": "typescript-plugin-css-modules" }]
+    "plugins": [{ "name": "typescript-plugin-css-modules" }],
+    "strictNullChecks": true,
+    "noUncheckedIndexedAccess": true
```

Comment:
needs better description of why you're enabeling this.

Reply:
Could the documentation do? I could just link `https://www.typescriptlang.org/tsconfig#noUncheckedIndexedAccess`

Reply:
both =)